ha/monitor/k8s/object_monitor.py

```diff
@@ -167,3 +172,50 @@ def run(self):
                 break
         Log.info(f"Stopping the {self.name}...")
 
+    def _is_published_alert(self, alert) -> bool:
+        """
+        Check incoming alert is already published or not.
+        If incoming alert is not found in published alerts, then
+        it is a new alert to publish. Alert will be stored like,
+        self._published_alerts = {
+            resource_name: { generation_id : alert }
```

Comment:
For disk alert, generation id will not be there (will be something else may be). So, how disk alerts will be stored?

Reply:
With the new commit, 
`self._published_alerts = { alert_key : alert }`
I handled this case,
```
        if incoming_generation_id:
            alert_key = f"{incoming_resource_type}_{incoming_generation_id}"
        else:
            alert_key = f"{incoming_resource_type}_{incoming_resource_name}"
```
Host alert also doesn't have generation id. But resource name is unique there.
In case Disk alert won't have any unique value, then we need to try other way.
At this time this is sufficient.

Reply:
`Note: Disk monitoring is not covered by k8s monitor`
However similar case suits for host alert.

Done.

Reply:
OK..Yes..Got it.